doc/source/whatsnew/v1.2.0.rst

```diff
@@ -349,6 +349,7 @@ Groupby/resample/rolling
 - Bug in :meth:`DataFrame.groupby` does not always maintain column index name for ``any``, ``all``, ``bfill``, ``ffill``, ``shift`` (:issue:`29764`)
 - Bug in :meth:`DataFrameGroupBy.apply` raising error with ``np.nan`` group(s) when ``dropna=False`` (:issue:`35889`)
 - Bug in :meth:`Rolling.sum()` returned wrong values when dtypes where mixed between float and integer and axis was equal to one (:issue:`20649`, :issue:`35596`)
+- Bug in :class:`pd.Grouper` now propagates ``dropna`` argument correctly (:issue:`36620`)
```

Comment:
i would remove this for now; in the followup which actually changes behavior you can add what is changed. this doesn't do anything per-se.

Reply:
Done